miasm2/os_dep/common.py

```diff
@@ -60,9 +60,15 @@ def alloc(self, jitter, size):
         @jitter: a jitter instance
         @size: the size to allocate
         """
+        return self.vm_alloc(jitter.vm, size)
 
+    def vm_alloc(self, vm, size):
```

Comment:
You may add the associated page rights as an argument, with default value `PAGE_READ | PAGE_WRITE`